Makefile

```diff
@@ -23,8 +23,6 @@ upgrade: $(COMMON_CONSTRAINTS_TXT)  ## update the requirements/*.txt files with
 
 extract_translations: ## extract strings to be translated, outputting .po files
 	cd $(WORKING_DIR) && i18n_tool extract
-	mv $(EXTRACTED_DJANGO) $(EXTRACTED_TEXT)
-	tail -n +20 $(EXTRACTED_DJANGOJS) >> $(EXTRACTED_TEXT)
-	rm $(EXTRACTED_DJANGOJS)
-	sed -i'' -e 's/nplurals=INTEGER/nplurals=2/' $(EXTRACTED_TEXT)
-	sed -i'' -e 's/plural=EXPRESSION/plural=\(n != 1\)/' $(EXTRACTED_TEXT)
+	mv $(EXTRACTED_DJANGO_PARTIAL) $(EXTRACTED_DJANGO)
```

Comment:
### Potential complexity

Just to confirm i18n_tools produces both of `django.po` and `django-partial.po`? It seems a bit odd to me.

There seem to be a bit complexity here which can be error prone, I think the following should be fine:

```suggestion
	msgcat $(EXTRACTED_DJANGO_PARTIAL) -o $(EXTRACTED_DJANGO)
```

Reply:
Thanks for keeping an eye on this @OmarIthawi No, it produces `django-partial.po` and `djangojs-partial.po`. We're overwriting `django.po` with the new messages produced by `django-partial.po`,,, then we combine `djangojs-partial.po` into `django.po`.